### PR TITLE
Added argument support and gitignore

### PR DESCRIPTION
**Pull request by NagaKanaparthy**
Added Args support so it could just be ran without needing to edit variables

### CODE REVIEW

**Review by krrish94 — CHANGES_REQUESTED** [review]
Hi @NagaKanaparthy,

Thanks for the PR, looks neat! Could you please update the `README` with the new commands to invoke the script too? I can merge to `master` once that's taken care of.

**Comment by NagaKanaparthy** [comment]
Hi @krrish94 , I finished the requested change please review.

File: .gitignore [hunk 1]
```diff
@@ -45,3 +45,4 @@ luac.out
 *.hex
 # End of https://www.gitignore.io/api/lua
 valid.txt
+model.t7
```
Comment (krrish94) [inline .gitignore]: Can you remove this line (and the above one too)?

**Review by krrish94 — CHANGES_REQUESTED** [review]
Almost there, just need to delete two lines. Many thanks for your efforts!

**Review by krrish94 — APPROVED** [review]
LGTM, merging!

File: .gitignore (hunk 1 above)
Comment (NagaKanaparthy) [reply .gitignore]: Done!

**Pull request merged**